src/map/conquest_data.cpp

```diff
@@ -0,0 +1,131 @@
+#include "conquest_data.h"
+
+#include "common/logging.h"
+#include "common/sql.h"
+#include "conquest_system.h"
+
+ConquestData::ConquestData()
+: regionControls(std::vector<region_control_t>(256))
+, influences(std::vector<influence_t>(256))
+{
+}
+
+void ConquestData::load(std::unique_ptr<SqlConnection>& sql)
```

Comment:
Given that this doesn't have any dependencies apart from the sql handle, I'd move all of this logic into the constructor and use the global sql handle with extern. I think its OK to lean on that design pattern for now, and at some point in the future I'll kill all the global extern usage of the sql handle etc.

Reply:
Also, given how little this data is, we could just lazy-load it just before its needed the first time?

Reply:
I'll lazy load it

Reply:
actually this is in map. which global should we rely on ? the map one?

Reply:
Done. On construction